cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/CompilationDatabase.kt

```diff
@@ -62,10 +62,19 @@ class CompilationDatabase : ArrayList<CompilationDatabase.CompilationDatabaseEnt
             return includePaths.keys.toList()
         }
 
+    fun addIncludePath(srcFile: File, paths: List<String>) {
+        includePaths[srcFile] = paths
+    }
+
     /** Returns the include paths for the specified file. */
     fun getIncludePaths(file: File): List<String>? {
         return includePaths[file]
     }
+
+    /** Returns the include paths for all files in compilation database. */
+    fun getIncludePaths(): List<String> {
```

Comment:
Can we rename this to `getAllIncludePaths` to not clash with the `includePaths` property. Also this can be converted to a Kotlin property:
```kotlin
val allIncludePaths: List<String> {
get() {
  return includePaths.values.flatten()
}
```